### ISSUE TITLE
Compiler errors in Sierra

### ISSUE THREAD

**Issue by Matt633** [idmean/CodaLess#6]
Just wanted to let you know that Coda has been crashing while using CodaLess, specifically when I compile Less with errors. I upgraded to Mac OS Sierra earlier this week, and now every time I compile and there's an error, I get a spinning wheel and have to Force Quit out of Coda. I never get to the error modal. 

Compiling without errors works fine. Thanks!

**Comment by idmean** [comment]
Thanks for your report. I’ll get back to this ASAP but I’m pretty busy at the moment to please bear with me.

**Comment by Matt633** [comment]
Of course - we definitely appreciate your help!

Thanks,
Matt


Matt Davis  //  Bistro Studios  //  bistrostudios.com <http://www.bistrostudios.com/>

**Comment by idmean** [comment]
I’ve upgraded the code to the latest Swift and macOS versions and had no troubles using the plugin afterwards. Can you please verify whether this version solves your issues? (Download from master branch)

**Comment by Matt633** [comment]
Thanks again - I downloaded the new plugin, and also reinstalled less and clean css. The issue is still occurring. Upon a complier error, I'm getting the dialogue box once or twice, and then on the third error, I'm getting a perpetual spinning wheel. I've also been in contact with Panic Inc - I will forward this thread to support.

**Comment by idmean** [comment]
Can you post the stack trace (if you get one)? If it’s related to CodaLess Panic’s support won’t be of much help, I guess.

**Comment by Matt633** [comment]
No, but I’m going to connect you directly. Hopefully this works — I’ve copied Panic support on this email. 

I’ve been speaking to Jesus Diaz. 

Thanks,
Matt
